scripts/docs/markdown/generateProps.js

```diff
@@ -0,0 +1,19 @@
+import stringOfLength from "../stringOfLength";
+import generateProp from "./generateProp";
+
+export default function generateProps(props) {
+  const title = "Props";
+
+  if (!props) {
+    return "";
+  }
+
```

Comment:
maybe picky, but perhaps move the `if !props, return` to the top to avoid the `const title = "Props";` from being defined

Reply:
_unless need the title there anyway_